styles/pages/TorchboxAcademy.module.scss

```
@@ -0,0 +1,18 @@
+.container {
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+.textContainer {
+    margin: 0 auto $grid * 3 auto;
+    max-width: 755px;
```

```suggestion
    max-width: $content--narrow;
```
Would be good to update the other textContainer classes with this as well.